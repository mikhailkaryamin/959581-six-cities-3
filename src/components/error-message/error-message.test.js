import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from './error-message.jsx';

test(`Error message`, () => {
  const tree = renderer.create(
      <ErrorMessage
        errorCode={401}
        onResetError={()=> {}}
      />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
