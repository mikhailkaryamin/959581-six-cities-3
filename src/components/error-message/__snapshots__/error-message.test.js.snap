// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error message 1`] = `
<div
  className="error-message"
  style={
    Object {
      "backgroundColor": "rgba(255, 99, 71, 0.75)",
      "borderRadius": "5px",
      "color": "#ffffff",
      "left": "50%",
      "padding": "25px",
      "position": "fixed",
      "right": "50%",
      "top": "5%",
      "width": "250px",
      "zIndex": "50",
    }
  }
>
  <div
    className="error-message__header"
    style={
      Object {
        "paddingBottom": "10px",
      }
    }
  >
    <b>
      Код ошибки: 
      401
    </b>
  </div>
  <div
    className="message__text"
  >
    <p>
      Что-то пошло не так:(
    </p>
  </div>
  <button
    className="error-message__button"
    onClick={[Function]}
    style={
      Object {
        "backgroundImage": "url(./img/close.svg)",
        "backgroundPosition": "center",
        "backgroundRepeat": "no-repeat",
        "backgroundSize": "10px",
        "color": "#ffffff",
        "height": "10px",
        "padding": "10px",
        "position": "absolute",
        "right": "5px",
        "top": "5px",
        "width": "10px",
        "zIndex": "10",
      }
    }
  />
</div>
`;
